server/broadlistening/pipeline/steps/hierarchical_aggregation.py

```
@@ -240,5 +350,38 @@ def _build_property_map(
+            if value is not None:
+                if isinstance(value, np.integer):
+                    value = int(value)
+                elif isinstance(value, np.floating):
+                    value = float(value)
+                elif isinstance(value, np.ndarray):
+                    value = value.tolist()
+                else:
+                    # Convert any other types to string to ensure serialization
+                    try:
+                        value = str(value)
+                    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)